cpplint.py

```diff
@@ -4371,11 +4371,13 @@ def CheckBraces(filename, clean_lines, linenum, error):
             '{ should almost always be at the end of the previous line')
 
   # An else clause should be on the same line as the preceding closing brace.
-  if re.match(r'\s*else\b\s*(?:if\b|\{|$)', line):
+  if last_wrong := re.match(r'\s*else\b\s*(?:if\b|\{|$)', line):
```

Comment:
`:=` syntax only works with python > 3.8, yet I am not aware of us dropping 3.7 support for the next release. So this change should be breaking CI (and it breaks my local env, too). if it does not break CI, then CI needs fixing.

Reply:
I see, it's the purpose of this MR> still tox.ini still references py3.7. Seems CI does not use tox.ini

Reply:
We can discuss here and change that if we don't think we should break 3.7 support (I pinged @jayvdb to try and discuss this last time but he didn't respond). Personally I think it should be fine as 3.7 is EOL.

Reply:
Also, there is a mismatch in tox.ini for py3.10 and py3.11

Reply:
#279